src/controllers/admin/profile.js

```diff
@@ -1,9 +1,26 @@
 const { join } = require('path');
+const getQuery = require('../../database/queries/admin/getQuery');
 
-exports.get = (req, res) => {
-  res.render(join('admin', 'profile.hbs'), {
-    css: [join('partials', 'adminNav'), join('partials', 'adminSidebar')],
-    js: ['domUyils', join('admin', 'sidebar')],
-    layout: 'admin',
-  });
+exports.get = (request, response) => {
+  const fullName = request.cookies.name;
+  let totalDonation = 0;
+  getQuery('admins', {
+    full_name: fullName,
+  })
```

Comment:
@Jamalat-shamallakh I would recommend storing (and searching) for users based on their ID instead of their full name.  It's probably fine for this project, but for future projects or larger projects, here's why storing ID is a little better than name:
1) There can be more than one user in your database with the same name
2) Sometimes users want the ability to change their full name, on their existing account.  For example, people change their name in facebook after they get married.

Reply:
thanks  for  your comment i will considered  it in the next time